### PR TITLE
Add vert5erini train scripts

### PR DESCRIPTION
**Pull request by MXueguang**
Add vert5erini train scripts

### CODE REVIEW

**Comment by MXueguang** [comment]
@ronakice updated.
The repl result is slightly better than in the paper

File: experiments/vert5erini/prepare_lp_input.py [hunk 1]
```diff
@@ -24,6 +24,6 @@
     for doc_id, indices in selection['evidence'].items():
         evidence = ' '.join(["sentence{}: ".format(idx+1) + corpus[int(doc_id)]['abstract'][i].strip() for idx, i in enumerate(indices)])
         format3_dev_ids.write("{}\t{}\n".format(claim_id, doc_id))
-        format3_dev.write("hypothesis: {} {}\n".format(claim, evidence))
+        format3_dev.write("query: {} {} relevant:\n".format(claim, evidence))
```
Comment (ronakice) [inline experiments/vert5erini/prepare_lp_input.py:27]: why is this changed?

File: experiments/vert5erini/prepare_lp_train_fever_input.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            if label == "true":
+                pos_counter += 1
+            elif label == 'false':
+                neg_counter += 1
+            elif label == "weak":
+                weak_counter += 1
+t5_input.close()
+print(f"Pos: {pos_counter}")
+print(f"Neg: {neg_counter}")
+print(f"Weak: {weak_counter}")
```
Comment (ronakice) [inline experiments/vert5erini/prepare_lp_train_fever_input.py:96]: File isn't used right?

**Comment by ronakice** [comment]
@X

**Review by ronakice — APPROVED** [review]
LGTM! Go ahead and merge :)
